orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationModuleConfig.java

```diff
@@ -84,4 +86,54 @@ public OrchestrationModuleConfig withMaskingConfig(
 
     return withMaskingConfig(newMaskingConfig);
   }
+
+  /**
+   * Adds input content filters to the orchestration configuration.
+   *
+   * <p>Preferred over {@link #withFilteringConfig(FilteringModuleConfig)} for adding input filters.
+   *
+   * @param contentFilters one or more content filters to apply to the input.
+   * @return a new {@code OrchestrationModuleConfig} instance with the specified input filters
+   *     added.
+   */
+  @Nonnull
+  public OrchestrationModuleConfig withInputFiltering(
+      @Nonnull final ContentFilter... contentFilters) {
```

Comment:
the spec declares minItems: 1, so we should enforce at least one value here, same as in the masking example